wordpress-mobile/WordPress-iOS
# Update Gutenberg to version 1.15.3

Fixes #

To test:
 - Using an iOS 13.2 device
 - Create a new GB post
 - Add one of the following blocks to it: List, Quote or Image captions
 - Try to write multiple lines
 - See if the block size updates correctly when typing or deleting Enters

Update release notes:
